app/src/main/res/layout/activity_main.xml

```diff
@@ -16,9 +16,10 @@
 
     <Button
         android:id="@+id/button_main_parking_size"
-        style="@style/ButtonConfigureParkingSize"
+        style="@style/Button"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:text="@string/button_configure_parking_size"
         app:layout_constraintTop_toBottomOf="@+id/main_toolbar"/>
```

Comment:
not use "+" to contraint the views. you only need to call :  id/main_toolbar

Reply:
done